### PR TITLE
Fix docs problems

### PR DESCRIPTION
**Pull request by maroux**
- update current version
- fix settings import so api.rst renders correctly
- a few problems found in review

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -13,24 +13,26 @@
...
-copyright = '2018, Aniruddha Maru'
-author = 'Aniruddha Maru'
+copyright = '2018, Automatic Labs'
+author = 'Automatic Labs'
 
 # The short X.Y version
 version = '1.0'
 # The full version, including alpha/beta/rc tags
-release = '1.0.2'
+release = '1.0.3-dev'
```
Comment (mngo87) [inline docs/conf.py]: remove `dev`

File: docs/conf.py [hunk 2]
```diff
@@ -92,7 +94,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```
Comment (mngo87) [inline docs/conf.py]: remove?

File: docs/conf.py (hunk 1 above)
Comment (maroux) [reply docs/conf.py]: Nope, it's intentional - `dev` is dropped when it's released (e.g. see `taskhawk/__init__.py`)

File: docs/conf.py (hunk 2 above)
Comment (maroux) [reply docs/conf.py]: This is a config file that contains lots (all?) the options with enabled options as commented out, so I decided to keep it that way.

**Pull request merged**